.github/workflows/gcp-deploy.yaml

```diff
@@ -80,62 +79,62 @@ jobs:
       HASURA_PORT: ${{env.HASURA_PORT}}
       FRONTEND_PORT: ${{env.FRONTEND_PORT}}
       HASURA_SECRET: ${{env.HASURA_SECRET}}
```

Comment:
Can you update `gcp-deploy-frontend.yaml` as well please?

Reply:
Done. To the best of my ability. We won't know 100% that it's correct until it deploys `develop`.